lib/Doctrine/DBAL/Schema/AbstractSchemaManager.php

```diff
@@ -209,6 +212,20 @@ public function listTableNames()
         return $this->filterAssetNames($tableNames);
     }
 
+    /**
+     * Returns a list of all tables in the current database.
+     *
+     * @return string[]
+     */
+    public function listAllTableNames()
```

Comment:
Let's make this method signature more changeable (`final`) and modern (`iterable`):
```suggestion
    final public function listAllTableNames(): iterable
```